### PR TITLE
Fix archives list styles and add tabs for message and run archives

### PR DESCRIPTION
**Pull request by norkans7**
Fixes https://github.com/rapidpro/rapidpro/issues/1370

Archives 2020-10-29 12-50-49

### CODE REVIEW

File: templates/archives/archive_list.haml [hunk 1]
```diff
@@ -33,20 +33,22 @@
   {% endrender %}
   {{page_top}}
 
--block top-form
-  - block search-form
+
+
+- block content
```
Comment (rowanseymour) [inline templates/archives/archive_list.haml:38]: `-block`

**Comment by nicpottier** [comment]
This looks a bit weird to me @ericnewcomer do you have ideas for a better treatment or places for inspiration there?

**Comment by ericnewcomer** [comment]
No great ideas this morning, but something like this is at least more consistent with the rest of the site. Feelings?

Screen Shot 2020-10-30 at 11 39 19 AM

**Comment by rowanseymour** [comment]
I feel that using a big blue button as subsection navigation is unfortunately consistent and I really look forward to rethinking navigation in the next release!

**Review by rowanseymour — APPROVED** [review]

**Comment by ericnewcomer** [comment]
Agreed!

**Comment by ericnewcomer** [comment]
Actually, we have places where these are light, and as a navigation option instead of an action, I think that makes more sense.

**Comment by ericnewcomer** [comment]
Screen Shot 2020-10-30 at 11 57 44 AM
Screen Shot 2020-10-30 at 11 58 18 AM

**Pull request merged**